config/example/node.env

```diff
@@ -63,9 +63,8 @@ NEOFS_CONTRACTS_PROXY=ad7c6b55b737b696e5c82c85445040964a03e97f
 NEOFS_MORPH_DIAL_TIMEOUT=30s
 NEOFS_MORPH_DISABLE_CACHE=true
 NEOFS_MORPH_RPC_ENDPOINT_0_ADDRESS="wss://rpc1.morph.fs.neo.org:40341/ws"
-NEOFS_MORPH_RPC_ENDPOINT_0_PRIORITY=2
 NEOFS_MORPH_RPC_ENDPOINT_1_ADDRESS="wss://rpc2.morph.fs.neo.org:40341/ws"
-NEOFS_MORPH_RPC_ENDPOINT_1_PRIORITY=1
+NEOFS_MORPH_RPC_ENDPOINT_1_PRIORITY=2
```

Comment:
Let's add priority to the `ENDPOINT_0` explicitly too

Reply:
added zero priority to the `0` endpoint